day_7/fizzbuzz.js

```diff
@@ -0,0 +1,18 @@
+for (i = 1; i <= 100; i++) {
+
+  if(i % 15 == 0) {
```

Comment:
I would definitely leave some breathing room between the if and the (

Reply:
@damwhit Fixed!